benchmark/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@babel/benchmark",
+  "private": true,
+  "type": "module",
+  "devDependencies": {
+    "@babel-baseline/generator": "npm:@babel/generator@7.14.5",
+    "@babel-baseline/helper-validator-identifier": "npm:@babel/helper-validator-identifier@7.10.4",
+    "@babel-baseline/parser": "npm:@babel/parser@7.14.5",
+    "@babel/generator": "workspace:*",
+    "@babel/helper-validator-identifier": "workspace:*",
+    "@babel/parser": "workspace:*",
```

Comment:
Now we can compare performances between any versions of babel packages by revising this `package.json`.

Reply:
what's the perf diff between ESBuild and Babel? Equal in performance?